src/components/PeopleFilters.tsx

```diff
@@ -1,27 +1,50 @@
-export const PeopleFilters = () => {
+/* eslint-disable @typescript-eslint/indent */
+import { FC } from 'react';
+import { CENTURIES_FILTER } from '../constants/centuries';
+import { SearchLink } from './SearchLink';
+import { useSearchParams } from 'react-router-dom';
+import classNames from 'classnames';
+import { SEX_FILTERS } from '../constants/sexFilter';
+
+interface PeopleFiltersProps {
+  updateQuery: (query: string) => void;
+}
+
+export const PeopleFilters: FC<PeopleFiltersProps> = ({ updateQuery }) => {
+  const [searchParams] = useSearchParams();
+  const existingCenturies = searchParams.getAll('centuries') || [];
+  const activeSex = searchParams.getAll('sex') || '';
+  const query = searchParams.get('query')?.toString();
+
   return (
     <nav className="panel">
       <p className="panel-heading">Filters</p>
 
       <p className="panel-tabs" data-cy="SexFilter">
-        <a className="is-active" href="#/people">
-          All
-        </a>
-        <a className="" href="#/people?sex=m">
-          Male
-        </a>
-        <a className="" href="#/people?sex=f">
-          Female
-        </a>
+        {Object.entries(SEX_FILTERS).map(([name, value]) => (
+          <SearchLink
+            key={name}
+            params={{ sex: value }}
+            className={classNames({
+              'is-active':
+                activeSex.includes(value || '') ||
```

Comment:
do u need it? u are already have it above 
`const activeSex = searchParams.getAll('sex') || '';`
```suggestion
                activeSex.includes(value) ||
```